activerecord/lib/active_record/connection_adapters/abstract/schema_creation.rb

```diff
@@ -101,6 +102,7 @@ def visit_CreateIndexDefinition(o)
           sql << "USING #{index.using}" if supports_index_using? && index.using
           sql << "(#{quoted_columns(index)})"
           sql << "WHERE #{index.where}" if supports_partial_index? && index.where
+          sql << "INCLUDE (#{quoted_include_columns(index.include)})" if supports_index_include? && index.include
```

Comment:
Is it possible to handle this in the postgres schema_creation file (activerecord/lib/active_record/connection_adapters/postgresql/schema_creation.rb) instead since this is a postgres only feature?

Reply:
Yes, I've moved it and the associated methods to the postgres `schema_creation`  file. I also moved `quoted_include_columns_for_index` to the postgres `schema_statements` file.